README.md

```diff
@@ -192,7 +193,47 @@ Start a port-forward to logs minio service:
 kubectl -n logs port-forward service/minio 8080
 ```
 
-Logs should appear in bucket test123
+Logs should appear in bucket `test123`.
+
+## Scaling
+
+A custom [KEDA](https://keda.sh/) external scaler
+[cwm-keda-external-scaler](https://github.com/iamazeem/cwm-keda-external-scaler)
+is being used to perform scaling.
+
+Make sure that the KEDA has already been deployed before proceeding with a
+`ScaledObject`. Use [install with YAML](https://keda.sh/docs/2.1/deploy/#yaml)
+method.
+
+By default, the external scaler is disabled i.e. no scaling.
+
+To enable it, use a custom `.values.yaml` and deploy accordingly:
+
+```yaml
+minio:
+  # ...
+  externalscaler:
+    enabled: true
+```
+
+Deploy: `helm upgrade -f .values.yaml --install cwm-worker-deployment-minio ./helm`
+
+The external scaler should be up and running.
+Now, the `ScaledObject` can be configured and deployed.
+
+```yaml
+minio:
+  # ...
+  scaledobject:
+    enabled: true
+```
+
+A custom `ScaledObject` has been provided for the minio deployment. It can be
+modified and deployed as required. It can be found under `./helm/templates/`
+directory.
```

Comment:
we need to be able to modify the scaled object via the helm values, you could allow to provide the relevant yaml as a value